__init__.py

```diff
@@ -578,7 +577,7 @@ def startSearch(self, value):
 def heartbeat_timer():
     log.info('sending thangs heartbeat')
     amplitude.send_amplitude_event("Thangs Blender Addon - Heartbeat", event_properties={'device_os': str(
-        fetcher.devideOS), 'device_ver': str(fetcher.deviceVer), 'source': "blender"})
+        fetcher.devideOS), 'device_ver': str(fetcher.deviceVer), 'source': "blender", 'addon_version': str(bl_info["version"])})

```

Comment:
Let's go ahead and clean things up a bit by sending `addon_version` in every amplitude event.  You should be able to assign it as a new field on the thangs_events class itself, and then let that class send it in with every amplitude event.